src/main/java/ch/njol/skript/expressions/ExprTimePlayed.java

```diff
@@ -53,7 +53,7 @@ public class ExprTimePlayed extends SimplePropertyExpression<OfflinePlayer, Time
 	private static final boolean IS_OFFLINE_SUPPORTED = Skript.methodExists(OfflinePlayer.class, "getStatistic", Statistic.class);
 
 	static {
-		register(ExprTimePlayed.class, Timespan.class, "time played", "offlineplayers");
+		register(ExprTimePlayed.class, Timespan.class, "(time (played|online)|(online|played) time)", "offlineplayers");
```

Comment:
> Thank you for your contribution :zap:
> 
> `Time played` is different than `play[ed] time`, the latter is for current playing time (since player joined aka `online time`, as you just suggested) therefore this is an invalid change. You may change this PR to add `online time` instead.

Explaining the above from the last PR, `time played` returns the total play time of a player in a server while `(play|online) time` should return the current play (online) time of a player since they last joined the server, therefore changing the syntax alone is invalid, you need to implement the functionality to get the player's online time.
![image](https://github.com/SkriptLang/Skript/assets/20037329/99eb1a0a-dd2d-4082-865f-c22c01a425ed)

Reply:
i get online time but shouldn't play time return the total played time? I'm pretty sure thats what it's always referred to as

Reply:
Play time being the same as time played also makes sense to me. I would not expect it to return the session time.

Reply:
let me know what you guys settle on, I won't be home for a bit so I won't be pushing anything until then anyway

Reply:
> Play time being the same as time played also makes sense to me. I would not expect it to return the session time.

I say:

Play time = time played = total time played in the server across all sessions

Online time = session time 
The pattern would look something like `(online time|session duration)`

Let me know what you think

Reply:
I wouldn't mind but it sounds different to my ears as I used to use /playtime command in quite different servers to show my current online time for different events and prizes. It's not a big deal but I would appreciate more opinions on this to settle it well. Also we can add an optional `[total]` to the time played pattern but that doesn't affect any results.

Reply:
Would `current/total` time played be okay? We could force it to be specified if we think it's going to be ambiguous.

Reply:
Yes we can, but it will be a breaking change

Reply:
This syntax should not reflect the current login time at all. You would use
`difference between now and last login` not total current play time. That does not sound like a good idea.

Reply:
Well, `online time` can not go into this expression as it's totally different than `total time played`. If this PR would be just a syntax improvement then we can keep it only to adding `play[ed] time`.

We should definitely add `online time` somewhere later instead of using `different between now and last login of player` as an enhancement.